data/mods/blazemod/modinfo.json

```diff
@@ -4,10 +4,9 @@
     "ident": "blazemod",
     "name": "Vehicle Additions Pack",
     "authors": [ "Blaze-whatdoyoumeanitsalreadytaken" ],
-    "maintainers": [ "Blaze-whatdoyoumeanitsalreadytaken" ],
+    "maintainers": [ "SouP" ],
```

Comment:
This is an array and you can honor the original creator too.

Reply:
Maintainers isn't for honoring people, it's for presenting facts. SouP is maintaining this, blaze hasn't for years.

Reply:
As long as he's in the author field I think that's credit enough, it just confuses people if he's listed as an active maintainer. If he comes back to work on it we can change it again.